rate-providers/registry.json

```
@@ -182,6 +182,25 @@
             "warnings": [],
             "factory": "",
             "upgradeableComponents": []
+        },
+        "0xc7177B6E18c1Abd725F5b75792e5F7A3bA5DBC2c": {
+            "name": "TruMaticRateProvider",
+            "summary": "safe",
+            "review": "./TruMaticRateProvider.md",
+            "warnings": [
+                "donation"
+            ],
+            "factory": "",
+            "upgradeableComponents": [
+                {
```

We should include the rate provider itself since that's upgradeable.
```suggestion
                {
                    "entrypoint": "0x4b697C8c3220FcBdd02DFFa46db5a896ae7a0843",
                    "implementationReviewed": "0x6e8135b003F288aA4009D948e9646588861C5575"
                },
```